### ISSUE TITLE
Incapable of setting new variable in Make Object Properties Private

### ISSUE THREAD

**Issue by vmmnnumava** [freeCodeCamp/freeCodeCamp#15501]
Challenge [Make Object Properties Private](https://www.freecodecamp.com/challenges/make-object-properties-private) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```javascript

var Car = function() {
  // this is a private variable
  var speed = 10;

  // these are public methods
  this.accelerate = function(change) {
    speed += change;
  };

  this.decelerate = function() {
    speed -= 5;
  };

  this.getSpeed = function() {
    return speed;
  };
};

var Bike = function() {

  // Only change code below this line.
var gear = 0;
  
};

this.setGear = function(val){
  gear += val;
};

this.getGear = function(){
  return gear;
};


var myCar = new Car();

var myBike = new Bike();

```

**Comment by erictleung** [comment]
@vmmnnumava Thank you for reporting this issue. There are multiple issues with your code. You prematurely close your `Bike` function and your function methods aren't in the `Bike` function. Also, I'd take a look at your `setGear` method once more.

This is a standard message notifying you that this issue seems to be a request for help. Instead of asking for help here, please click the "help" button on the challenge on freeCodeCamp, which will take you to the help chatroom for that specific challenge. You can also view our [full list of official chatrooms](http://forum.freecodecamp.com/t/free-code-camp-official-chat-rooms/19390).

If you think I'm wrong in closing this issue, please reopen it and add further clarification. Thank you, and happy coding!

**Comment by vmmnnumava** [comment]
@erictleung  Thank you very much. I was able to complete the exercise thanks to your help, but when i click the "help" button I can´t type in the chatroom and ask for help. A message appears saying"Only GitHub users can join this room." and i can´t type. I created a Github account but I still can´t type. 

As i am new to this programming "world" and English is not my mother language , learning to programming is not as intuitive as maybe an native english speaker. If you were able to help me fix this issue or maybe point me someone who is able and happy to help me I would be very glad.

Once again ,thank you very much for your help.
